scripts/upgrade.sh

```diff
@@ -32,7 +32,4 @@ docker volume rm prover-node-docker_workspace-volume
 docker volume rm prover-node-docker_rocksdb_data
 
 # Remove the image and re-pull the latest image
-docker image rm zkwasm:latest
-
-# Rebuild image locally
-DOCKER_BUILDKIT=0 docker build --rm --network=host -t zkwasm .
\ No newline at end of file
+docker image rm rhaoio/prover-node-dev:latest
```

Comment:
I think we can also move the
`docker image rm zkwasm:latest`
into stop.sh try_stop_zkwasm_mongodb()?
So we can help them to delete image as they do not need it any more?